Jenkinsfile

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env groovy
+
+@Library('sec_ci_libs@v2-latest') _
+
+def master_branches = ["master", ] as String[]
+def release_branches = ["master", ] as String[]
+
+pipeline {
+  agent {
+    dockerfile {
+      args  '--shm-size=1g'
+    }
+  }
+  options {
+    timeout(time: 3, unit: 'HOURS')
```

Comment:
Can we go down to 1 here? Even that is insanely high :)

Reply:
15m 👍